### ISSUE TITLE
Make where clause optional

### ISSUE THREAD

**Issue by tracker1** [kshvmdn/fsql#8]

**Comment by kshvmdn** [comment]
This is implemented as of a00c275084be77b781b7b4d5b088ab5eba3a1670. It's on a non-master branch though, so until it gets merged, you'll have to build manually to test it.

If you'd like to do that, here's how --

```sh
$ git clone https://github.com/kshvmdn/fsql.git $GOPATH/src/github.com/kshvmdn/fsql
$ cd $_
$ git fetch origin
$ git checkout -b refactor/query-structure origin/refactor/query-structure
$ go install .
$ which fsql
$GOPATH/bin/fsql
```

**Comment by kshvmdn** [comment]
Merged as of a6c3339668ca477722d16238c1cf2f6e1616955d.

Will keep the issue open until confirmation.

**Comment by tracker1** [comment]
Working...
